docs/ui/interactions/fragments/web/chatbot/usage.md

```diff
@@ -125,13 +125,9 @@ _App.vue_
 
 </docs-filter>
 
-<inline-fragment framework="react" src="~/ui/interactions/fragments/web/chatbot/properties-jsx.md"></inline-fragment>
+<ui-component-props tag="amplify-chatbot" prop-type="attr" use-table-headers></ui-component-props>
 
-<inline-fragment framework="vue" src="~/ui/interactions/fragments/web/chatbot/properties-attributes.md"></inline-fragment>
```

Comment:
Why the removals and will this be present on all frameworks now?

Reply:
I manually wrote the chatbot properties because https://github.com/aws-amplify/docs/pull/2432 wasn't merged at the time. Now I'm replacing them with auto documentation now 👍